server/src/portfolio/cron.js

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import cron from 'node-cron'
+import { pgPool } from './db/dbClient.js';
+import { updatePortfolioValue } from './portfolio.js';
+
+
+export const schedulePortfolioCron = () => {
+    cron.schedule("0 * * * *", () => {
+        console.log('running a task every hour');
+        const dbClient = await pgPool.connect();
+        updatePortfolioValue(dbClient);
+        
```

Comment:
you should add a `try{} finally {} ` block where you would call `dbClient.release()` in the `finally` block. We need to release pool clients to prevent exhaustion.